jubatus/core/fv_converter/char_splitter.cpp

```diff
@@ -28,136 +33,33 @@ void char_splitter::split(
     const std::string& string,
     std::vector<std::pair<size_t, size_t> >& ret_boundaries) const {
   std::vector<std::pair<size_t, size_t> > bounds;
+  const jubatus::util::data::string::ustring target = string_to_ustring(string);
 
   size_t last = 0;
   while (true) {
-    size_t begin = find_first_not_of(string,separators_, last);
+    size_t begin = target.find_first_not_of(separator_, last);
     if (begin == std::string::npos) {
       break;
     }
 
-    size_t end = find_first_of(string,separators_, begin);
+    size_t begin_bytes = ustring_to_string(target.substr(0, begin)).size();
+    size_t end = target.find_first_of(separator_, begin);
     if (end == std::string::npos) {
       size_t len = string.size() - begin;
-      bounds.push_back(std::make_pair(begin, len));
+      size_t len_bytes = ustring_to_string(target.substr(begin, len)).size();
```

Comment:
I think this ``target.substr(begin, len)`` is equal to ``target.substr(begin)`` because this code get the substring of ``target`` from ``begin`` to the end of ``target``. 
And also the variable ``len`` is not necessary in this condition.